src/lib/utilities/history-summary-plucker.ts

```diff
@@ -0,0 +1,93 @@
+import { findValueIfItIncludesWord } from './find-value-if-it-includes-word';
+
+const summaryExtractors = {
+  ActivityTaskCancelRequested: (d) => ({ Id: d.activityId }),
+  ActivityTaskCompleted: (d) => ({ result: d.result }),
+  ActivityTaskFailed: (d) => ({
+    details: d.details,
+    reason: d.reason,
+  }),
+  ActivityTaskScheduled: (d) => ({
+    'Close Timeout': d.startToCloseTimeout,
+    Id: d.activityId,
+    input: d.input,
+    Name: d.activityType.name,
+  }),
+  ActivityTaskStarted: (d) => ({
+    attempt: d.attempt,
+    identity: d.identity,
+    requestId: d.requestId,
+  }),
+  ActivityTaskTimedOut: (d) => ({
+    'Timeout Type': d.failure?.timeoutFailureInfo?.timeoutType,
```

Comment:
What would the default value for this be? If the `?.` fails it will have undefined as a value so you'll return `{"Timeout Type": undefined}` not sure what the consuming code is like. Will that be okay?